src/code-server/install.sh

```diff
@@ -53,14 +53,32 @@ if [[ "$DISABLEWORKSPACETRUST" == "true" ]]; then
     DISABLE_FLAGS+=(--disable-workspace-trust)
 fi
 
+CERT_FLAGS=()
+
+if [[ -n "$CERT" ]]; then
+    CERT_FLAGS+=(--cert)
+    CERT_FLAGS+=("$CERT")
```

Comment:
Suggestion: Alternative way:

```suggestion
    CERT_FLAGS+=(--cert "$CERT")
```